echopype/echodata/zarr_combine.py

```diff
@@ -830,10 +854,11 @@ def _write_append_dims(
                     compute=True,
                     storage_options=storage_options,
                     synchronizer=zarr.ThreadSynchronizer(),
+                    consolidated=consolidated,
```

Comment:
You should not add consolidated here. The metadata has already been created at this point and you are recomputing it.